examples/gallery/plot/points-categorical.py

```diff
@@ -0,0 +1,59 @@
+"""
+Color points by categories
+---------------------------
+The :meth:`pygmt.Figure.plot` method can be used to plot symbols which are
+color-coded by categories.
+"""
+
+import numpy as np
+import pandas as pd
+import pygmt
+
+# Load sample penguins data and convert 'species' column to categorical dtype
+df = pd.read_csv("https://github.com/mwaskom/seaborn-data/raw/master/penguins.csv")
+df["species"] = df.species.astype(dtype="category")
+
+# Use pygmt.info to get region bounds (xmin, xmax, ymin, ymax)
+# The below example will return a numpy array like [2.  4.4 4.3 7.9]
```

Comment:
```suggestion
# The below example will return a numpy array like [2.0, 4.4, 4.3, 7.9]
```

Reply:
I noticed that the shown limits still belong to the iris data set, made an update.